CMakeLists.txt

```diff
@@ -344,7 +345,9 @@ if(ENABLE_ASM)
 		endif()
 		add_definitions(-DHAVE_GNU_STACK)
 	elseif(APPLE AND "${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "x86_64")
-		set(HOST_ASM_MACOSX_X86_64 true)
+		# disable for now (.section .rodata invalid)
+		# set(HOST_ASM_MACOSX_X86_64 true)
```

Comment:
Ran into this just a few days ago. Try this please:https://boringssl-review.googlesource.com/c/boringssl/+/57445/4/crypto/perlasm/x86_64-xlate.pl